src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetCPInfo.cs

```diff
@@ -19,6 +18,6 @@ internal unsafe struct CPINFO
         }
 
         [DllImport(Libraries.Kernel32)]
-        internal static extern unsafe int GetCPInfo(uint codePage, CPINFO* lpCpInfo);
+        internal static extern int GetCPInfo(uint codePage, out CPINFO lpCpInfo);
```

Comment:
`out` instead of pointer is not an improvement here. It just makes the PInvoke stub more expensive.

And avoiding `unsafe` keyword for PInvokes has no real value. PInvokes are implicitly unsafe code.

Reply:
I've updated code to match existing coding style, will undo the change